src/assets/locales/fr.json

```diff
@@ -53,13 +74,17 @@
   "button.cancel": "Annuler",
   "button.continue": "Continuer",
   "button.next": "Suivant",
+  "chat-message-input.warning-max-characters": "Caractères max",
```

Comment:
For easier review, I give the English version in comment. Here the English is "Max characters".
In the code src/react-components/room/ChatSidebar.js, it's "Max characters" (${messageLength}/${maxLength})" when you reach the maximum characters allowed.